lib/sync_checker/formats/edition_base.rb

```diff
@@ -190,6 +192,10 @@ def latest_edition_is_rejected?(document)
           document.pre_publication_edition &&
           document.pre_publication_edition.rejected?
       end
+
+      def disallowed_http_codes
+        (400..599)
```

Comment:
I don't think we should worry about it looking odd, at the moment it is something that is constant to our check so I think we should express it using that syntax.